### DISCUSSION TITLE
Turns to early

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4364] [General]
Hi developers,

I think it is a know issue. Aircraft is turning much to early. I am using custom FBW. And also turning with more than 25 degrees bank angle ... hmmmm Isn´t this to much in normal law?

![grafik](https://user-images.githubusercontent.com/76531893/114180308-53557c00-9940-11eb-913b-beacdc1ada7b.png)

Greets Klaus

**Comment by CptBayer** [comment]
Hi,

I think the issue with following the flightpath in NAV mode is not new. For example on my flight to SBFZ with a VOR DME approach on RWY 13 the autopilot didn´t follow the path to or in the near of the VOR FLZ. NAV mode switched in the picture below much to early to waypoint FORT (strange wp). Fix CF13 was ignored completely. So I disconnected the NAV mode and was using HDG mode for the rest of the approach (before I flew manually)....

In the correct way, NAV mode should fly towards VOR FLZ, then turning into the teardrop procedure for FIX CF13. 

As you also can see, a dashed line from a point that wasn´t on the path after switching to HDG mode. Strange behaviour too. Normally the dashed line should be under my plane on the way to VOR FLZ.

![grafik](https://user-images.githubusercontent.com/76531893/114713604-97c28c80-9d31-11eb-8ec3-dcba16cc6d75.png)

Greets Klaus

**Reply by robinottawa** [reply]
So maybe the early turn is actually a general disconnection problem? I get disconnected every flight at somewhere near to ILS approach. I use the DIR button to get back onto the flight plan and that seems to work. I find the turns are just right as long as, of course, I am on the FP. Cheers

**Comment by CptBayer** [comment]
another example... ignoring a lot of waypoints and drawing a line from the last not ignored wp to VYDOA ... 

![grafik](https://user-images.githubusercontent.com/76531893/115421865-a4952380-a1fc-11eb-9823-d83f31e787a9.png)
